api/tests/acceptance/application/users/update-user-details-for-administration_test.js

```diff
@@ -0,0 +1,145 @@
+const { expect, databaseBuilder, generateValidRequestAuthorizationHeader, insertUserWithRolePixMaster } = require('../../../test-helper');
+
+const createServer = require('../../../../server');
+
+describe('Acceptance | Controller | users-controller-update-user-details-for-administration', () => {
+
+  let server;
+  let user;
+  let options;
+
+  beforeEach(async () => {
+    server = await createServer();
+
+    user = await insertUserWithRolePixMaster();
+
+  });
+
+  describe('Error case', () => {
+
+
```

Comment:
C'est plutôt cette ligne en trop qui fait planter la CI @octo-pito : 20:1  error  More than 1 blank line not allowed  no-multiple-empty-lines

Reply:
Mais pourquoi ça passe en local ?

Reply:
J'ai corrigé et pushé